src/server.py

```diff
@@ -433,50 +434,38 @@ def run_seta_query():
     return {'failures': failures}
 
 
-@app.route("/data/setasummary/")
-@json_response
-def run_seta_summary_query():
-    query = session.query(Seta.date).distinct().all()
-    retVal = {}
-    dates = []
-    for d in query:
-        dates.append(d[0].strftime("%Y-%m-%d"))
-
-    for d in dates:
-        count = session.query(Seta.id).filter(Seta.date == d).count()
-        retVal['%s' % d] = "%s" % int(count)
-
-    return {'dates': retVal}
-
-
 @app.route("/data/setadetails/")
 @json_response
 def run_seta_details_query():
-    startDate, date = clean_date_params(request.args)
-    active = sanitize_bool(request.args.get("active", 0))
+    inactive = sanitize_bool(request.args.get("inactive", 0))
```

Comment:
oh!  you are right.  This is only used for supporting buildbot legacy api!  I will add a comment.